File: src/main/scala/org/openapitools/server/model/OfferCardsContentStatusDTO.scala
package org.openapitools.server.model


/**
 * Список товаров с информацией о состоянии карточек.
 *
 * @param offerCards Страница списка товаров с информацией о состоянии карточек. for example: ''null''
 * @param paging  for example: ''null''
*/
final case class OfferCardsContentStatusDTO (
  offerCards: Seq[OfferCardDTO],
  paging: Option[ForwardScrollingPagerDTO] = None
)

File: ympa_fsharp_functions_server/src/model/OfferCardsContentStatusDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.ForwardScrollingPagerDTO
open ympa_fsharp_functions_server.Model.OfferCardDTO

module OfferCardsContentStatusDTO =

  //#region OfferCardsContentStatusDTO

  [<CLIMutable>]
  type OfferCardsContentStatusDTO = {
    [<JsonProperty(PropertyName = "offerCards")>]
    OfferCards : OfferCardDTO[];
    [<JsonProperty(PropertyName = "paging")>]
    Paging : ForwardScrollingPagerDTO;
  }

  //#endregion
